products/caasp/main.pm

```diff
@@ -121,7 +121,9 @@ sub load_feature_tests {
     loadtest 'caasp/overlayfs';
     loadtest 'caasp/services_enabled';
     loadtest 'caasp/one_line_checks';
-    loadtest 'caasp/nfs_client' if get_var('NFS_SHARE');
+    if (get_var('NFS_SHARE') && !is_caasp('local')) {
```

Comment:
Is there a chance to set the variable properly in the test suites? I guess we also could unset in in main.pm. WDYT?

Reply:
It's set in test suites - but it's used only when you do `client isos post...`
When you clone job then your local setup is ignored, so you have to `clone_job... NFS_SHARE=0` - and I always forget, so after 30m I have to clone again properly :)

Do you mean something like `set_var('NFS_SHARE', 0) unless is_caasp('local')` somewhere in main.pm?

Reply:
Yes, exactly that. We have some other similar issues, get_required_var helps to get some clue, but it doesn't help in scenarios like this one.